GitTfs/Core/IGitTfsRemote.cs

```diff
@@ -38,7 +38,6 @@ public interface IGitTfsRemote
         string TfsUsername { get; set; }
         string TfsPassword { get; set; }
         IGitRepository Repository { get; set; }
-        [Obsolete("Make this go away")]
```

Comment:
That's what I thought at a moment but it's a huge work that probably not worth it...